tests/MetricTest.php

```diff
@@ -2,6 +2,7 @@
 namespace SiteMaster\Plugins\Unl;
 
 use SiteMaster\Core\Auditor\Parser\HTML5;
+use \SiteMaster\Plugins\Unl\Metric as UNLMetric;
```

Comment:
Not sure why this was added. You're already in this namespace _and_ using this class on 103.